neo/Network/P2P/Payloads/NetworkAddressWithTime.cs

```diff
@@ -29,7 +27,7 @@ public static NetworkAddressWithTime Create(IPEndPoint endpoint, ulong services,
         void ISerializable.Deserialize(BinaryReader reader)
         {
             Timestamp = reader.ReadUInt32();
-            Services = reader.ReadUInt64();
+            Services = (VersionServices)reader.ReadUInt64();
```

Comment:
@erikzhang, versionservice still need to be cast to `ulong`? 
It is now a static `0x01`?

Reply:
Yes, is needed because is a flag (ulong)